capz/run-capz-e2e.sh

```diff
@@ -136,9 +136,15 @@ run_e2e_test() {
         curl -L -o /tmp/kubernetes-test-linux-amd64.tar.gz https://storage.googleapis.com/k8s-release-dev/ci/"${CI_VERSION}"/kubernetes-test-linux-amd64.tar.gz
         tar -xzvf /tmp/kubernetes-test-linux-amd64.tar.gz
 
-        export GINKGO_FOCUS=${GINKGO_FOCUS:-"\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-scheduling\].SchedulerPreemption"}
-        export GINKGO_SKIP=${GINKGO_SKIP:-"\[LinuxOnly\]|\[Serial\]|\[Slow\]|\[Excluded:WindowsDocker\]|Networking.Granular.Checks(.*)node-pod.communication|Guestbook.application.should.create.and.stop.a.working.application|device.plugin.for.Windows|Container.Lifecycle.Hook.when.create.a.pod.with.lifecycle.hook.should.execute(.*)http.hook.properly|\[sig-api-machinery\].Garbage.collector"}
-        export GINKGO_NODES="${GINKGO_NODES:-"4"}"
+        if [[ ! "$RUN_SERIAL_TESTS" == "true" ]]; then
```

Comment:
```suggestion
        if [[ ! "${RUN_SERIAL_TESTS:-}" == "true" ]]; then
```

to avoid `./capz/run-capz-e2e.sh: line 139: RUN_SERIAL_TESTS: unbound variable`